### PR TITLE
Switch references from libpod.conf to containers.conf

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Should help with https://github.com/containers/podman/issues/6982

**Comment by TomSweeneyRedHat** [comment]
tests are falling down @rhatdan

File: docs/source/markdown/podman.1.md [hunk 1]
```diff
@@ -217,11 +217,11 @@ the exit codes follow the `chroot` standard, see below:
 
 ## FILES
 
-**libpod.conf** (`/usr/share/containers/libpod.conf`)
+**containers.conf** (`/usr/share/containers/containers.conf`)
 
-    libpod.conf is the configuration file for all tools using libpod to manage containers, when run as root.  Administrators can override the defaults file by creating `/etc/containers/libpod.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/libpod.conf` is created and replaces some fields in the system configuration file.
+    containers.conf is the configuration file for container engines.  Administrators can override the defaults file by creating `/etc/containers/containers.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/containers.conf` is created and replaces some fields in the system configuration file.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman.1.md:222]:
```suggestion
    containers.conf is the configuration file for container engines, when run as root.  Administrators can override the defaults file by creating `/etc/containers/containers.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/containers.conf` is created and replaces some fields in the system configuration file.
```

Comment (rhatdan) [reply docs/source/markdown/podman.1.md]: @TomSweeneyRedHat How about the new wording.

File: docs/source/markdown/podman-attach.1.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ or name, either to view its ongoing output or to control it interactively.
 You can detach from the container (and leave it running) using a configurable key sequence. The default
 sequence is `ctrl-p,ctrl-q`.
 Configure the keys sequence using the **--detach-keys** option, or specifying
-it in the **libpod.conf** file: see **libpod.conf(5)** for more information.
+it in the **containers.conf** file: see **containers.conf(5)** for more information.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-attach.1.md:18]: Probably should add container.conf(5) to the bottom of this page.

File: docs/source/markdown/podman-events.1.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ podman\-events - Monitor Podman events
 
 Monitor and print events that occur in Podman. Each event will include a timestamp,
 a type, a status, name (if applicable), and image (if applicable).  The default logging
-mechanism is *journald*. This can be changed in libpod.conf by changing the `events_logger`
+mechanism is *journald*. This can be changed in containers.conf by changing the `events_logger`
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-events.1.md:13]: Probably should add container.conf(5) to the bottom of this page.

File: docs/source/markdown/podman.1.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ created by the other.
 
 **--cgroup-manager**=*manager*
 
-CGroup manager to use for container cgroups. Supported values are cgroupfs or systemd. Default is systemd unless overridden in the libpod.conf file.
+CGroup manager to use for container cgroups. Supported values are cgroupfs or systemd. Default is systemd unless overridden in the containers.conf file.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman.1.md:26]:
```suggestion
The cgroup manager to use for container cgroups. Supported values are cgroupfs or systemd. Default is systemd unless overridden in the containers.conf file.
```

File: docs/source/markdown/podman.1.md [hunk 5]
```diff
@@ -217,11 +217,13 @@ the exit codes follow the `chroot` standard, see below:
...
 
-    libpod.conf is the configuration file for all tools using libpod to manage containers, when run as root.  Administrators can override the defaults file by creating `/etc/containers/libpod.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/libpod.conf` is created and replaces some fields in the system configuration file.
+    Podman has builtin defaults for command line options. These defualts can be overridden using the containers.conf configuration files.
 
-    Podman uses builtin defaults if no libpod.conf file is found.
+Distributions ship the `/usr/share/containers/containers.conf` file with their defaults. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specify fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution files, which override the builtin defaults.
+
+    Podman uses builtin defaults if no containers.conf file is found.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman.1.md:226]: space/tab issue on this line?

File: docs/source/markdown/podman.1.md [hunk 6]
```diff
@@ -217,11 +217,13 @@ the exit codes follow the `chroot` standard, see below:
...
+**containers.conf** (`/usr/share/containers/containers.conf`)
 
-    libpod.conf is the configuration file for all tools using libpod to manage containers, when run as root.  Administrators can override the defaults file by creating `/etc/containers/libpod.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/libpod.conf` is created and replaces some fields in the system configuration file.
+    Podman has builtin defaults for command line options. These defualts can be overridden using the containers.conf configuration files.
 
-    Podman uses builtin defaults if no libpod.conf file is found.
+Distributions ship the `/usr/share/containers/containers.conf` file with their defaults. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specify fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution files, which override the builtin defaults.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman.1.md:224]:
```suggestion
Distributions ship the `/usr/share/containers/containers.conf` file with the default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify the defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the fields from these files, if they exist. Fields specified in the user's file override the administrator's file, which overrides the distribution's file, which override the built-in defaults.
```

Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman.1.md:224]: tweaked slighly @rhatdan, but I like where you're going with it.

File: docs/source/markdown/podman-attach.1.md (hunk 2 above)
Comment (rhatdan) [reply docs/source/markdown/podman-attach.1.md:18]: Done

File: docs/source/markdown/podman-events.1.md (hunk 3 above)
Comment (rhatdan) [reply docs/source/markdown/podman-events.1.md:13]: Done

File: docs/source/markdown/podman.1.md (hunk 5 above)
Comment (rhatdan) [reply docs/source/markdown/podman.1.md:226]: Fixed

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL again.

**Comment by csomh** [comment]
The following `podman` man-pages still contain references to `libpod.conf` without mentioning `containers.conf`. If the intent is to replace `libpod.conf` with `containers.conf`, then probably these should be also updated:

```
$ git grep -le 'libpod\.conf' docs/source/markdown/podman-*
docs/source/markdown/podman-attach.1.md
docs/source/markdown/podman-create.1.md
docs/source/markdown/podman-events.1.md
docs/source/markdown/podman-run.1.md
docs/source/markdown/podman-system-migrate.1.md
docs/source/markdown/podman-system-renumber.1.md
```

Furthermore there is a mention of `libpod.conf` in `docs/tutorials/rootless_tutorial.md`, too.

File: docs/source/markdown/podman.1.md [hunk 7]
```diff
@@ -217,11 +217,13 @@ the exit codes follow the `chroot` standard, see below:
...
+**containers.conf** (`/usr/share/containers/containers.conf`)
 
-    libpod.conf is the configuration file for all tools using libpod to manage containers, when run as root.  Administrators can override the defaults file by creating `/etc/containers/libpod.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/libpod.conf` is created and replaces some fields in the system configuration file.
+    Podman has builtin defaults for command line options. These defaults can be overridden using the containers.conf configuration files.
 
-    Podman uses builtin defaults if no libpod.conf file is found.
+Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specify fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution's file, which override the built-in defaults.
```
[the remark this answers is not in the document]
Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman.1.md:224]:
```suggestion
Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges it's builtin defaults with the specified fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution's file, which override the built-in defaults.
```

[the remark this answers is not in the document]
Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman.1.md:224]: specify to specified in the prior, doesn't show well in the diff

**Comment by rhatdan** [comment]
@csomh I think I got them all.

File: docs/source/markdown/podman.1.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (rhatdan) [reply docs/source/markdown/podman.1.md]: Fixed

File: docs/source/markdown/podman.1.md [hunk 8]
```diff
@@ -217,11 +217,13 @@ the exit codes follow the `chroot` standard, see below:
...
+**containers.conf** (`/usr/share/containers/containers.conf`)
 
-    libpod.conf is the configuration file for all tools using libpod to manage containers, when run as root.  Administrators can override the defaults file by creating `/etc/containers/libpod.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/libpod.conf` is created and replaces some fields in the system configuration file.
+    Podman has builtin defaults for command line options. These defaults can be overridden using the containers.conf configuration files.
 
-    Podman uses builtin defaults if no libpod.conf file is found.
+Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges its builtin defaults with the specified fields from these files, if they exist. Fields specified in the users file override the administrators file, which overrides the distribution's file, which override the built-in defaults.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman.1.md:224]: sorry if I missed this this morning, missing a possesive adminstrator.
```suggestion
Distributions ship the `/usr/share/containers/containers.conf` file with their default settings. Administrators can override fields in this file by creating the `/etc/containers/containers.conf` file.  Users can further modify defaults by creating the `$HOME/.config/containers/containers.conf` file. Podman merges its builtin defaults with the specified fields from these files, if they exist. Fields specified in the users file override the administrator's file, which overrides the distribution's file, which override the built-in defaults.
```

**Comment by rhatdan** [comment]
@mheon @csomh @QiWang19 @vrothberg @baude @jwhonce @ashley-cui PTAL

**Comment by QiWang19** [comment]
LGTM. @TomSweeneyRedHat  PTAL

**Pull request merged**